lib/registerTypes/RegisterTypes.h

```diff
@@ -201,6 +201,7 @@ void registerTypesMapObjects2(Serializer &s)
 	//s.template registerType<CObstacleInstance>();
 		s.template registerType<CObstacleInstance, MoatObstacle>();
 		s.template registerType<CObstacleInstance, SpellCreatedObstacle>();
+		s.template registerType<CObstacleInstance, BattlefieldLimiterObstacle>();

```

Comment:
Yep - what is the current solution for such things? No register new classes at all?